src/interfaces/loan-interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface ILoan {
```

Comment:
Personal Opinion, I m not fan of the `I` in front just to say is Interface, can be just `Loan`

Reply:
do you have any special reason?  I could learn from you